parsl/dataflow/dflow.py

```diff
@@ -479,17 +479,20 @@ def _add_input_deps(self, executor, args, kwargs):
 
         inputs = kwargs.get('inputs', [])
         for idx, f in enumerate(inputs):
-            if isinstance(f, File) and f.is_remote():
-                inputs[idx] = f.stage_in(executor)
+            if isinstance(f, File):
+                if f.is_remote():
+                    inputs[idx] = f.stage_in(executor)
 
-        for kwarg, f in kwargs.items():
-            if isinstance(f, File) and f.is_remote():
-                kwargs[kwarg] = f.stage_in(executor)
+        for kwarg, potential_f in kwargs.items():
+            if isinstance(potential_f, File):
```

Comment:
What is this change from `and` to 2x `if` for?

Reply:
This is just an artifact of adding and removing code from the data flow manager, I'll change it make in the next commit